packages/hop-node/src/watchers/HealthCheckWatcher.ts

```diff
@@ -129,6 +133,22 @@ type InvalidBondWithdrawal = {
   timestamp: number
 }
 
+type UnrelayedTransfers = {
```

Comment:
This entity type should be named as singular and not plural: `UnrelayedTransfers` => `UnrelayedTransfer`